content/docs/09.administrator-guide/01.configuration/03.enterprise-edition/multi-tenancy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Multi-tenancy configuration
+---
+
+For an overview of what is multi-tenancy, please read the [Architecture - Multi-tenancy](../../../08.architecture.md#multi-tenancy) documentation.
+
+## Configuring multi-tenancy
+
+### `kestra.ee.tenants.enabled`
+
+Set it to `true` to enable multi-tenancy.
+By default multi-tenancy is disabled.
+
+### `kestra.ee.tenants.default-tenant`
+
+The default tenant is a tenant without identifier (or the null tenant), it exists for backward compatibility when multi-tenancy is enabled in an existing Kestra instance. If you disable the default tenant in a Kestra instance that already have flows and executions, those will not be available anymore to anyone.
+
+When multi-tenancy is enabled in a new Kestra instance, it is advised to disabled the default tenant so all tenants will have an identifier.
+
+By default, the default tenant is enabled, set it to `false` to disable it.
```

Comment:
```suggestion
By default, multi-tenancy is disabled, and the default tenant is set to `true`. Once you enable multi-tenancy, you can set the **default tenant** to `false` to disable it so that your Kestra instance includes only the tenants you explicitly create.
```